### PR TITLE
v1.16 ref2fec-centroid documentation

### PR DESCRIPTION
**Pull request by parkerduckworth**
Add the `ref2vec-centroid` page for the v1.16.0 release

### CODE REVIEW

File: developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+An object whose class is configured to use ref2vec-centroid will have its vector calculated (or recalculated) as a result of these events:
+- Creating the object with references already assigned as properties 
+- Updating an existing object's list of references. Note that this can happen several ways:
+  - Object `PUT`: update all of the object's properties, containing a different set of references. This totally replaces the object's existing reference list with the newly provided one
+  - Object `PATCH`: update an existing object by adding any newly provided reference(s) to the object's existing reference list
+  - Reference `POST`: create a new reference to an existing object
+  - Reference `PUT`: update all of the object's references
+- Deleting references from the object. Note that this can happen several ways:
+  - Object `PUT`:update all of the object's properties, removing all references
```
Comment (aliszka) [inline developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md:141]: `PUT`:update - missing space

**Review by aliszka — APPROVED** [review]
good job!
I like ants example :)

File: developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+...
+```
+
+# How to configure
+
+​In your Weaviate schema, you must define how you want this module to vectorize your data. If you are new to Weaviate schemas, you might want to check out the [getting started guide on the Weaviate schema](../getting-started/schema.html) first.
+
+For example, here is an `Article` class which is configured to use ref2vec-centroid. This module requires only a class-level `moduleConfig`, containing two fields:
+
+1. `referenceProperties`: a list of the class' reference properties which should be used during the calculation of the centroid.
```
Comment (dirkkul) [inline developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md:61]: "class'" is there an s missing?

**Review by dirkkul — APPROVED** [review]

File: developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md (hunk 2 above)
Comment (parkerduckworth) [reply developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md:61]: Both `Class's` and `Class'` are correct ways to express a singular possessive noun which ends in an `s`  🙂

File: developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md (hunk 1 above)
Comment (parkerduckworth) [reply developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md]: thanks, fixed

File: developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md (hunk 2 above)
Comment (dirkkul) [reply developers/weaviate/current/retriever-vectorizer-modules/ref2vec-centroid.md:61]: ha, you never stop learning :)

**Pull request merged**